packages/client/hmi-client/src/components/widgets/tera-related-documents.vue

```diff
@@ -5,7 +5,9 @@
 		</p>
 		<ul>
 			<li v-for="document in relatedDocuments" :key="document.id">
-				{{ document.name }}
+				<a @click="openAsset({ assetId: document.id!, pageType: AssetType.Documents })">{{
+					document.name
```

Comment:
@YohannParis I've made the tera-asset-link component for this.  I've also made a tera-asset-icon component since the way we were retrieving component was pretty tedious to be doing everwhere.
<img width="779" alt="Screenshot 2023-10-20 at 12 35 59 PM" src="https://github.com/DARPA-ASKEM/terarium/assets/33158416/90cecc52-e027-4871-a180-9f073e796b51">